tasklist/els-schema/src/main/java/io/camunda/tasklist/es/RetryElasticsearchClient.java

```diff
@@ -792,11 +792,14 @@ public boolean putLifeCyclePolicy(final PutLifecyclePolicyRequest putLifecyclePo
   }
 
   public GetLifecyclePolicyResponse getLifeCyclePolicy(
-      final GetLifecyclePolicyRequest getLifecyclePolicyRequest) {
-    return executeWithRetries(
-        String.format("Get LifeCyclePolicy %s ", getLifecyclePolicyRequest.getPolicyNames()),
-        () ->
-            esClient.indexLifecycle().getLifecyclePolicy(getLifecyclePolicyRequest, requestOptions),
-        null);
+      final GetLifecyclePolicyRequest getLifecyclePolicyRequest) throws IOException {
+    try{
```

Comment:
can we keep this try catch inside `executeWithRetries` ? it will retry on all the errors except the 404

Reply:
Good suggestion, I updated on https://github.com/camunda/camunda/pull/18903#:~:text=execution%20with%20retries-,fe80331,-Merge%20state